en/developer_guide/api_group_context.rst

```diff
@@ -0,0 +1,61 @@
+API group context
+=================
+
+Context
+-------
+
+To increase readability and reduce the code duplication in the API, a transformer will only be
+linked to one entity and one facade. This way, each time you will transform an entity into
+a facade, it will be done using the same parameters. In some case, you do not want to display
+all the data from a facade in a certain context.
+
+For instance, when you list all the nodes, the area and blocks might not be usefull.
+
+The ``JMSSerializerBundle`` allows you to hide some properties on the serialization by using
+some serialization group. The main issue with this method is that you will still have to
+complete all the data in the facade.
+
+We are going to see how we can use the Open Orchestra project to limit the amount of data set
+in the facade depending on the context
+
+Usage
+-----
+
+In a controller action, you can directly add an annotation :
+
+.. code-block:: php
+
+    /**
+     * @Api\Groups({"GROUP_FOO"})
+     */
+    public function fooAction()
+
+With the requirements :
+
+.. code-block:: php
+
+    use OpenOrchestra\ApiBundle\Controller\Annotation as Api;
+
+Then in the transformer, you can directly call the ``hasGroup`` method :
+
+.. code-block:: php
+
+    $this->hasGroup("GROUP_FOO");
+
+This method will check if the group has been added in the action annotation.
```

Comment:
Can you put a complete exemple of how it works ? Does the developper has to check the group to hydrate the attribute or to not hydrate it ?